index.html

```diff
@@ -2611,7 +2615,7 @@ <h3>Terms of Use</h3>
     "https://www.w3.org/2018/credentials/examples/v1"
   ],
   "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-  "type": ["VerifiablePresentation", "CredentialManagerPresentation"],
+  "type": ["VerifiablePresentation", "TermsOfUsePresentation"],
```

Comment:
I'd prefer that we don't put this in here as it's probably not going to be best practice to do so. That is, the spec already states that you can specify termsOfUse in a presentation, so no additional type is necessary here. We don't want people copy-pasting this example.

Reply:
I suggest adding some other funky extension to the presentation to make it more obvious (if we want to show this sort of thing in the specification).

Reply:
addressed via [12681f9](https://github.com/w3c/vc-data-model/pull/658/commits/12681f9a24bb7669e0f23268b16ef4d318b172c2)